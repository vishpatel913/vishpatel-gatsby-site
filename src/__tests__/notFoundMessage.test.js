import React from "react";
import renderer from "react-test-renderer";
import NotFoundMessage from "../components/notFoundMessage";

describe("NotFoundMessage", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<NotFoundMessage />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
