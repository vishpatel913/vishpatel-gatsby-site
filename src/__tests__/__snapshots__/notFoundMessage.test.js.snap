// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundMessage renders correctly 1`] = `
.c0 {
  text-align: center;
}

.c1 {
  width: 500px;
}

<div
  className="c0"
>
  <h1>
    NOT FOUND
  </h1>
  <img
    className="c1"
    src="test-file-stub"
  />
  <p>
    You just found a page that doesn't exist... oh the sadness.
  </p>
</div>
`;
